kotlin/perform-spatial-operations/src/main/java/com/esri/arcgisruntime/sample/performspatialoperations/MainActivity.kt

```diff
@@ -131,14 +130,13 @@ class MainActivity : AppCompatActivity() {
     }
 
     // create and add a green graphic to show input polygon 2
-    SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, -0x66ff6700, lineSymbol).let {
-      inputGeometryOverlay.graphics.add(Graphic(inputPolygon2, it))
-    }
+    val greenFill = SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, -0x66ff6700, lineSymbol)
```

Comment:
to be consistent with the above `SimpleFillSymbols`, should probably replace this hex code with `Color.GREEN`

Reply:
I found Color.GREEN a bit hard to look at, but I am not sure what is more important here. I am happy to make the change!